taccsite_cms/templates/nav_search.raw.html

```diff
@@ -3,9 +3,7 @@
           so Portal and User Guide can render this markup into their markup. -->
 
 <style type="text/css">
-  /* Hide search bar before all relevant styles have loaded */
-  /* FAQ: Manage visibility of search bar to avoid FOUC */
-  /* HACK: GH-101: Resolve via dynamically-added `[data-status="loaded"]` */
+  /* SEE: https://github.com/TACC/Core-Styles/blob/main/source/_imports/elements/tacc-search-bar.md */
```

Comment:
That _will_ be the URL. Until https://github.com/TACC/Core-Styles/pull/17, the actual URL is https://github.com/TACC/Core-Styles/blob/task/TUP-231-cms-css-from-styles-repo-to-cms-repo/source/_imports/elements/tacc-search-bar.md.